.gitignore

```diff
@@ -7,9 +7,9 @@ config.yml
 config.xml
 
 # Ignore VueJS compiled files
-views/css/app.css
-views/js/app.js
-views/js/chunk-vendors.js
+views/css
+views/js
+views/img
```

Comment:
It would be great to keep the previous gitignore, as we are versioning additional CSS and JS files